testdata/lint/main.go

```diff
@@ -0,0 +1,17 @@
+// Linter's testdata.
+package main
+
+import "log/slog"
```

Comment:
You forget about this:

> - Test import renaming `import "l" "log/slog". May need "pkg:" inside "forbid:".

Also please add calls using …Context functions and methods, to make sure linter will accept them.

All of this means you'll need to have 32 log lines in main() instead of current 8: extra 8 with ...Context functions/methods, and extra 16 using package `l` and variable `ll` (created using `ll := l.Default()`).

Reply:
> Test import renaming `import "l" "log/slog". May need "pkg:" inside "forbid:".
I tested this cases manually, works correct.
I'll add them.